instructions/docs/multi-cmk.md

```diff
@@ -143,8 +148,29 @@ Try out combinations of Grant permissions for your application and watch how the
 TODO
 ```
 
-```bash tab="JavaScript Node.JS"
-TODO
+```javascript tab="JavaScript Node.JS"
```

Comment:
Should we throw in some comments to adding/removing grants from the CMKs and trying some of these operations again?

Reply:
I was thinking about this.
Maybe then the commands should interleave the steps?